dd-sdk-android/src/main/kotlin/com/datadog/android/rum/webview/RumWebViewClient.kt

```diff
@@ -26,6 +26,19 @@ import com.datadog.android.rum.RumResourceKind
  *
  * This will map the page loading, and webview errors into RUM Resource and
  * Error events respectively.
+ *
+ * This class is deprecated and you should use the new
+ * [com.datadog.android.webview.DatadogEventBridge] to track your WebViews:
+ *
+ * ```kotlin
+ * val configuration = Configuration.Builder().setWebViewTrackingHosts(listOf(<YOUR_HOSTS>))
+ * Datadog.initialize(this, credentials, configuration, trackingConsent)
+ *
+ * webView.webViewClient = WebViewClient()
+ * webView.webChromeClient = WebChromeClient()
```

Comment:
Same, those are unnecessary

Reply:
Actually they are because otherwise any time you navigate to a web page from inside the WebView page it will switch to `Chrome` application in a different process, this will make you loose all the events as there is no longer a bridge there.

Reply:
See more here : https://developer.android.com/guide/webapps/webview#HandlingNavigation

Reply:
Then you should add that to the documentation to explain why those are necessary, because we don't know what is the expected behavior that our customers want. Something like : 

```
// By default, link navigation will be delegated to a third party app. If you want all navigation to happen inside of the webview, uncomment the following line
// myWebView.webViewClient = WebViewClient()
```

Also this explains why you need the `WebViewClient`, but why do you also _need_ the `WebChromeClient`?

Reply:
Yeah, I was kind of assuming that people using WebView already know but you are right this is a bit weird and requires more details. Good question about the `WebChromeClient` , I will try without that to see if it works.